Memories/Memories/Views/CustomViews.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CustomViews.swift
+//  Test
+//
+//  Created by Alexander Supe on 13.03.20.
+//  Copyright © 2020 Alexander Supe. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable class RoundButton: UIButton {
+    @IBInspectable var cornerRadius: CGFloat = 10 { didSet {
+        self.layer.cornerRadius = cornerRadius
+        }}
+    
+}
```

Comment:
This should be in it's own file RoundButton.swift

Put the { on separate lines so it's easy to see open/close. It makes it hard to read.

```swift
@IBDesignable class RoundButton: UIButton {
    @IBInspectable var cornerRadius: CGFloat = 10 {
        didSet {
            self.layer.cornerRadius = cornerRadius
        }
    }
}
```